internal/backends/winit/Cargo.toml

```
@@ -21,7 +21,7 @@ path = "lib.rs"
 [features]
 wayland = ["winit/wayland", "glutin/wayland", "copypasta/wayland", "i-slint-renderer-skia?/wayland"]
 x11 = ["winit/x11", "glutin/x11", "glutin/glx", "copypasta/x11", "i-slint-renderer-skia?/x11"]
-renderer-winit-femtovg = ["femtovg", "fontdb", "libc", "yeslogic-fontconfig-sys", "winapi", "dwrote", "imgref", "unicode-script", "ttf-parser", "rgb", "i-slint-core/box-shadow-cache"]
+renderer-winit-femtovg = ["i-slint-renderer-femtovg"]
```

```suggestion
renderer-winit-femtovg = ["dep:i-slint-renderer-femtovg"]
```

But if we do that, we should make it consistent with other features

---

I don't feel very strong about this, but it sounds like something that should be applied to all crates. That said, I don't mind if we don't use dep: everywhere - at least so far I haven't been confused by the mix :)